rendercanvas/html.py

```diff
@@ -84,7 +84,7 @@ def buttons_mask_to_tuple(mask) -> tuple[int, ...]:
                     res += (mouse_button_map.get(i, i),)
             return res
 
-
+        self._pointer_inside = False # keep track for the pointer_move event
```

Comment:
You might want to look into:

* Pointer capturing
* Mouse leave and enter events

Maybe you already have and this comment is entirely moot